packages/hawtio/src/plugins/springboot/Health.tsx

```diff
@@ -24,12 +13,8 @@ import {
   QuestionCircleIcon,
 } from '@patternfly/react-icons'
 
-const componentSpanRecord: Record<string, number[]> = {
-  diskSpace: [7, 2],
-  ping: [5, 0],
-  camelHealth: [5, 1],
-  camel: [5, 1],
-}
+const HEALTH_COMPONENTS = ['diskSpace', 'camelHealth', 'camel']
```

Comment:
Would this benefit from a more explicit name. HEALTH_COMPONENTS seems a little ambiguous given all of the compoments have come from `loadHealth` in the springboot-service?